packages/kbn-es/src/cluster.ts

```diff
@@ -6,42 +6,49 @@
  * Side Public License, v 1.
  */
 
-const fs = require('fs');
-const fsp = require('fs/promises');
-const execa = require('execa');
-const chalk = require('chalk');
-const path = require('path');
-const Rx = require('rxjs');
-const { Client } = require('@elastic/elasticsearch');
-const { downloadSnapshot, installSnapshot, installSource, installArchive } = require('./install');
-const { ES_BIN, ES_PLUGIN_BIN, ES_KEYSTORE_BIN } = require('./paths');
-const {
+import fs from 'fs';
+import fsp from 'fs/promises';
+import chalk from 'chalk';
+import { resolve as pathResolve, relative as pathRelative } from 'path';
```

Comment:
How come this usage is not in line with the other node module usages? 
```
import fs from 'fs';
// but:
import { resolve as pathResolve } from 'path'
```

Reply:
Changing to `import * as path from 'path';`